<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_uart_getport">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>99</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Get port</string>
    </property>
    <property name="icon">
     <iconset theme="port">
      <normaloff>.</normaloff>.</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Quit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>410</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_uartport">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_uart_connect">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>99</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
    <property name="icon">
     <iconset theme="port">
      <normaloff>.</normaloff>.</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_write_start">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>99</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Writedata</string>
    </property>
    <property name="icon">
     <iconset theme="port">
      <normaloff>.</normaloff>.</iconset>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_textbox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>281</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_host_ip">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>000.000.000.000;_</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>562</x>
      <y>3</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Host IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>562</x>
      <y>34</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_host_user">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_host_user_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>562</x>
      <y>65</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Passwd</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
